### ISSUE TITLE
CLI installation script fails on Amazon Linux 2

### ISSUE THREAD

**Issue by rannn505** [configu/configu#322]
### Current Behavior

Configu CLI installation script (`curl https://cli.configu.com/install.sh | sh`) is failing on Amazon Linux 2 (EC2, Bastion machine)

### Expected Behavior

Installation script should work on AL2

### To Reproduce

1. create a machine with Amazon Linux 2 (ec2 machine)
2. run cli installation script
3. see error

### Context

<img width="855" alt="image" src="https://github.com/configu/configu/assets/11806915/0a643d6f-e3e2-474a-a7d9-ad8166d6705c">

**Comment by Sadaf-A** [comment]
Hey @rannn505 ! I would like to work on this please assign

**Comment by rannn505** [comment]
@Sadaf-A All yours brother :)

Do you have access to any aws account in order to pick up an ec2 machine with al2?
DA, if you produce a machine in the micro tier, it should be included in the free aws offer.

In addition, I would be happy if in your test you could produce another machine with a newer Amazon Linux (I read that version 2023 was released) and check whether the script works on it or not on this occasion.

Thanks in advance.

**Comment by Sadaf-A** [comment]
> @Sadaf-A All yours brother :)
> 
> Do you have access to any aws account in order to pick up an ec2 machine with al2?
> DA, if you produce a machine in the micro tier, it should be included in the free aws offer.
> 
> In addition, I would be happy if in your test you could produce another machine with a newer Amazon Linux (I read that version 2023 was released) and check whether the script works on it or not on this occasion.
> 
> Thanks in advance.

yes, I have an aws account I will let you know about any progress

**Comment by Sadaf-A** [comment]
@rannn505 the error was resolved however i ran into another error that said node is not installed we can make our own `AMI` with node pre installed to resolve this issue or we could add a user data script

**Comment by Sadaf-A** [comment]
```
/usr/local/lib/configu/bin/node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /usr/local/lib/configu/bin/node)
/usr/local/lib/configu/bin/node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /usr/local/lib/configu/bin/node)
Configu CLI installed to /usr/local/bin/configu
node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by node)
node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by node)
```
I'm getting this output finally after installing node

**Comment by rannn505** [comment]
@Sadaf-A this is very strange as the cli is compiled and shipped with node.js binary in it. can you try installing node.js 20.10 on the AL2 machine and let me know if it works?
also, when you said "resolved", what do you mean? 

> @rannn505 the error was resolved however i ran into another error that said node is not installed we can make our own `AMI` with node pre installed to resolve this issue or we could add a user data script

**Comment by Sadaf-A** [comment]
> @Sadaf-A this is very strange as the cli is compiled and shipped with node.js binary in it. can you try installing node.js 20.10 on the AL2 machine and let me know if it works? also, when you said "resolved", what do you mean?
> 
> > @rannn505 the error was resolved however i ran into another error that said node is not installed we can make our own `AMI` with node pre installed to resolve this issue or we could add a user data script

what I meant by error being resolved was that the `Your path is missing usr/local/bin, you need to add this to use this installer` error is not coming anymore
